src/main/java/org/takes/rq/multipart/RqMtFake.java

```diff
@@ -143,49 +147,60 @@ public InputStream body() throws IOException {
      * Fake body .
      * @since 0.33
      */
+    @RequiredArgsConstructor
     private static final class FakeBody implements Body {
 
         /**
-         * Content of the body.
+         * The parts.
          */
-        private final String content;
+        private final Request[] parts;
 
         /**
-         * Ctor.
-         * @param parts Fake request body partsFakeBody
-         * @throws IOException If fails
+         * The content.
+         */
+        private final AtomicReference<String> content = new AtomicReference<>();
```

Comment:
@andreoss Why not instantiate the variable in constructor ?

Reply:
@baudoliver7 Replaced with `Scalar` initialized in ctor.

Reply:
@andreoss Ok